tool/README.md

```diff
@@ -139,6 +106,21 @@ From the git GUI tool or from github.com directly:
    tool/tag_version.sh;
    ```
 
+### Verify and Submit the release notes
+
+See the release notes
+[README.md](https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/framework/release_notes/README.md)
+for details on where to add DevTools release notes to Flutter website and how to test them.
+
```

Comment:
Would it make sense to add header for the list? Like:

To verify and submit release notes:

Reply:
I believe that is already the case